README.rst

```diff
@@ -424,6 +428,82 @@ Keys to take special care are the ones needed to configure Kafka and advertised_
      - ``lowest``
      - Decides on what basis the Karapace cluster master is chosen (only relevant in a multi node setup)
 
+Karapace Schema Registry authorization file is an optional JSON configuration, which contains a list of authorized users in ``users`` list and a set of access control rules in ``permissions`` list.  Karapace uses HTTP Basic Authentication for user authentication.
```

Comment:
`list of access control rules`, not a set.

Better like this

which contains a list of authorized users in ``users`` and a list of access control rules in ``permissions``.

Reply:
> Karapace uses HTTP Basic Authentication for user authentication.

I don't think it's 100% clear from the documentation that Karapace uses this _only_ if the auth file is there. Also, the files is used only in Schema Registry side.

I think it'd be cleaner to dedicate a chapter to Karapace Schema Registry authentication and authorization to have these things in the correct context.

Reply:
It looks it's missing how exactly authorizations works. I don't think it's clear at all from the documentation.

We have this: 

> A regular expression used to match against accessed resource of form Subject:<subject_name> or Config:<optional_subject_name>

There are at least these things missing:

* How global ``Config:`` resource works
* Mentioning about content filtering, not just limiting access or allowing

Maybe this logic should be opened a little? Maybe not explaining each endpoint separately, but the general idea like if the user has been granted read access to a subject X, he has access to all "read" endpoints that have that subject in the URL and the endpoints that returns schemas will filter in the subject X. Etc.